core-backend/common/src/lib.rs

```diff
@@ -356,10 +373,10 @@ pub trait BackendState {
     }
 
     /// Process alloc function result
-    fn process_alloc_func_result<T: Sized, E: BackendAllocExtError>(
+    fn process_alloc_func_result<T: Sized, ExtAllocError: BackendAllocExternalitiesError>(
```

Comment:
<T, E> is also quite a stable pattern for generics returning `Result`.

Reply:
In this module we have another method, which accepts `Result` with a generic error type. Although `<T, E>` is a suitable pattern, I'd move from short generics in this crate/package as it lowers readability.

Reply:
agreed to resolve